### PR TITLE
Increased TextAppearanceSelector spacing and restyled its sliders (created Slider.svelte).

### PR DESCRIPTION
**Pull request by Caleb-Parks**
- Implemented a slider component for type=range input elements (Slider.svelte) and used it in the TextAppearanceSelector.
- Wrapped the contents of TextAppearanceSelector in a grid div to increase vertical spacing.

### CODE REVIEW

**Comment by chrisvire** [comment]
When I first click on the Text Appearance button, it looks like the first slider has focus?

image

**Comment by chrisvire** [comment]
I created issue #299 to track the first control having focus.

I fixed a few things during review:
* change barColor and progressColor to reactive statements
* fix text color when dark theme
* make icons larger now that there is more space

**Review by chrisvire — APPROVED** [review]
👍

File: src/lib/components/TextAppearanceSelector.svelte [hunk 1]
```diff
@@ -94,68 +98,72 @@ TODO
...
-                            on:click={() => ($theme = 'Normal')}
-                        />
-                    {/if}
-                    {#if themes.includes('Sepia')}
-                        <button
-                            class="dy-btn-sm"
-                            style:background-color={buttonBackground('Sepia')}
-                            style:border={buttonBorder('Sepia', $theme)}
-                            on:click={() => ($theme = 'Sepia')}
+            <div class="grid gap-4">
```
Comment (chrisvire) [inline src/lib/components/TextAppearanceSelector.svelte:101]: @Caleb-Parks: why is there an addition grid added?

Comment (Caleb-Parks) [reply src/lib/components/TextAppearanceSelector.svelte:101]: The grid div that's wrapping the others is to implement the extra spacing, per the second bullet point of my PR comments.

**Pull request merged**